android/build.gradle

```diff
@@ -35,9 +36,9 @@ android {
     lintOptions {
         abortOnError false
     }
-    compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_11
-        targetCompatibility = JavaVersion.VERSION_11
+    compileOptions {        
```

Comment:
Can you please remove those trailing spaces? 😃

Reply:
Oh, didnt see that. Im also fan of 100% clean code. I will commit an update ;)